chingu-coders/Voyage2-Turtles-02
# Master branch merge

## Purpose
* Merge dev branch into master for production

## Approach/Fixes
* Includes timer input changes, sticky note padding, quicklink adjustments, styling tweaks, and gulp refactor for development efficiency

## Testing
* Download this PR to your local machine and test extensively!
* Open quicklinks and add a really long link name to see that the name breaks apart appropriately to match the container
* Check that quicklinks open in same tab now instead of new tab
* Open up a sticky note and check that there is padding there
* Test the pomodoro timer and make sure it works for both work and break periods
* Go into the settings and try changing the break/work time periods with the slider *AND* the input (should highlight on hover) and check to make sure those values persist

## Notes
Please check and test everything very closely since this is a merge to our master branch to be updated on Chrome and Firefox!
